frontend/src/components/Loading.jsx

```diff
@@ -1,3 +1,32 @@
-const Loading = (props) => <div className="text-center">Loading...</div>
+import PropTypes from 'prop-types'
```

Comment:
There will be a merge conflict with this loading component, but I like your approach so include please it.

Reply:
Sure, I'll include this loading component. Thanks!